ts/features/it-wallet/store/reducers/itwCredentialsReducer.ts

```diff
@@ -63,4 +88,18 @@ export const ItwCredentialsPidSelector = (state: GlobalState) =>
     O.none
   );
 
+/**
+ * Selects the credentials stored in the wallet.
+ * @param state - the global state
+ * @returns the PID from the wallet.
```

Comment:
It returns the PID or the (q)eea?

Reply:
Nice catch, copy pasted it from above. Fixed.